package.json

```diff
@@ -384,6 +384,12 @@
 						],
 						"scope": "resource"
 					},
+					"asciidoc.preview.templates": {
+						"type": "string",
+						"default": "",
+						"markdownDescription": "%asciidoc.preview.templates.desc%",
+						"scope": "resource"
+					},
```

Comment:
Since `template_dirs` is an array, I think we should use a type array:

```
"asciidoc.preview.templates": {
  "type": "array",
  "markdownDescription": "%asciidoc.preview.templates.desc%",
  "items": {
    "type": "string"
  },
  "uniqueItems": false,
  "scope": "resource"
},
```

Reply:
fixed